### PR TITLE
Feat/graphql for services

### PR DESCRIPTION
**Pull request by Andrei0872**
This would solve #43.

Notes:
*   the `/posts/:id/report` has no tests on the BE so I skipped its graphql implementation as I have only followed the tests for `integration/routes/posts.js`. As a result, there is no GraphQL implementation for `/posts/:id/report` on the client.
* there was no need for vue-apollo as the fetching logic is done in the **content service**, not in the template. Thus, this PR contains, apart from the corresponding tests, replacements such as:
```typescript
http.get('/v1/posts/latest') ---> http.get('/graphql', { params: { query: latestPostsQuery }, { variables } })
```

### CODE REVIEW

File: packages/services/__tests__/content.ts [hunk 1]
```diff
@@ -8,11 +8,24 @@ import {expected as latestExpected, response as latestResponse} from './fixtures
...
+function encode(val: string) {
+    return encodeURIComponent(val).
+        replace(/%40/gi, '@').
+        replace(/%3A/gi, ':').
+        replace(/%24/g, '$').
+        replace(/%2C/gi, ',').
+        replace(/%20/g, '+').
+        replace(/%5B/gi, '[').
+        replace(/%5D/gi, ']');
+}
```
Comment (idoshamun) [inline packages/services/__tests__/content.ts]: What about using `encodeURI` instead of `encodeURIComponent` with all the manual regex replacements?

Comment (Andrei0872) [reply packages/services/__tests__/content.ts]: This is the code extracted from [axios](https://github.com/axios/axios/blob/master/lib/helpers/buildURL.js#L5-L14) in order to make the request work with `nock`, while writing tests. I'm not sure though why they decided to do the replacements... :sweat_smile:.

Comment (idoshamun) [reply packages/services/__tests__/content.ts]: Hmmm let's try `encodeURI`. I prefer to not keep such "magic" snippets if not necessary and if indeed necessary let's add a comment. Please check and let me know

Comment (Andrei0872) [reply packages/services/__tests__/content.ts]: Hi! Sorry for the delay! I just checked and it works fine with `encodeURI`. 
I'll add some comments and then I'll update the PR.

Comment (idoshamun) [reply packages/services/__tests__/content.ts]: Sounds legit. Let's keep it this way just add a comment explaining this :)

**Pull request merged**